model/bench_test.go

```diff
@@ -0,0 +1,16 @@
+package model
+
+import "testing"
+
+func BenchmarkString(b *testing.B) {
```

Comment:
IMHO, this code is just noise...
When a newbie join our team, he/she will be confused this function, why it exist?
Half years later, maybe you forget why yourself.

Last but not least, when you add a new file to the project, put license information please.

Reply:
+1
I would suggest removing this file, it's not useful in the future.

Reply:
Fixed.